x-pack/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -66,7 +66,7 @@ export function KueryBar(props: { prepend?: React.ReactNode | string }) {
 
   const example = examples[processorEvent || 'defaults'];
 
-  const { indexPattern } = useDynamicIndexPatternFetcher(processorEvent);
+  const { indexPattern } = useDynamicIndexPatternFetcher();
```

Comment:
Removed `processorEvent` from the dynamic index pattern. It barely had an impact since 850 fields of 905 are shared. So limiting to a single `processorEvent` didn't have any significant advantages.